test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+odoo-addon-base_rest @ git+https://github.com/OCA/rest-framework@refs/pull/225/head#subdirectory=setup/base_rest
```

Comment:
pls ALWAYS use  a separated commit for test-requirements w/ proper msg like "Add TMP test dependency"

Reply:
I forgot second time)))) Sure.